migtests/tests/pg-dependent-ddls/init-db

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+source ${SCRIPTS}/functions.sh
+
+echo "Creating ${SOURCE_DB_NAME} database on source"
+run_psql postgres "DROP DATABASE IF EXISTS ${SOURCE_DB_NAME};"
+run_psql postgres "CREATE DATABASE ${SOURCE_DB_NAME};"
+
+echo "Initialising source database."
+psql "postgresql://${SOURCE_DB_USER}:"${SOURCE_DB_PASSWORD}"@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${SOURCE_DB_NAME}" -f schema.sql
```

Comment:
Can you try using the following call instead?
```
psql_import_file schema.sql
```

Reply:
done